src/components/tl/ucp/tl_ucp.c

```diff
@@ -140,6 +140,12 @@ ucc_config_field_t ucc_tl_ucp_lib_config_table[] = {
      ucc_offsetof(ucc_tl_ucp_lib_config_t, allgather_kn_radix),
      UCC_CONFIG_TYPE_UINT},
 
+    {"ALLGATHER_USE_LOOPBACK", "0",
+     "If set to 1 performs network loopback for self copy, otherwise uses mc "
+     "cuda copy",
```

Comment:
```suggestion
     "If set to 1 performs network loopback for self copy, otherwise uses mc ",
```
it is not necessarily cuda